consts/consts.go

```diff
@@ -318,6 +318,9 @@ const ES_RESULT_TYPE_TAGS = "tags"
 const ES_RESULT_TYPE_BLOG_POSTS = "posts"
 const ES_RESULT_TYPE_TWEETS = "tweets"
 
+// Result of many tweets in one hit
+const SEARCH_RESULT_TWEETS_MANY = "tweets_many"
```

Comment:
This is being used in eval only?
Should also be used in serving?

Reply:
also used in search, see https://github.com/Bnei-Baruch/archive-backend/pull/124